### ISSUE TITLE
Terraform crash when refreshin the state - AWS WAF ACL with manually attached managed rule

### ISSUE THREAD

**Issue by pepsimin** [terraform-providers/terraform-provider-aws#4259]
### Terraform Version
Terraform v0.11.7

### Affected Resource(s)
Please list the resources as a list, for example:
- aws_wafregional_web_acl

### Panic Output
https://gist.github.com/pepsimin/9effc31cd015311c0ce51ea6adab31aa

### Expected Behavior
 - successful apply

### Actual Behavior
- terraform crash on plan or refresh

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
`terraform apply`
or
`terraform plan`
or
`terraform refresh`

### Important Factoids
The WAF ACL was manually modified in AWS to contain a managed rule, for which I believe terraform lacks support. Manually detaching the rule, then running terraform apply again will progress normally.

**Comment by bflad** [comment]
This should be fixed with #5053 which was just merged into master and will release with version 1.27.0 of the AWS provider, likely middle of this week. 👍 

```hcl
# Implementation is similar for aws_wafregional_web_acl resource
resource "aws_waf_web_acl" "example" {
 # ... other configuration ...
  rules {
    # ... other configuration ...
    override_action {
       type = "NONE"
    }
    type = "GROUP"
    rule_id = "${aws_waf_rule_group.example.id}"
  }
}
```

**Comment by bflad** [comment]
This has been released in [version 1.27.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1270-july-11-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
